integration_tests/integration/scan/manuallyEnterBarcode.spec.ts

```diff
@@ -18,8 +18,8 @@ context('Manual Barcode Entry Page', () => {
     Page.verifyOnPage(ManualBarcodeEntryPage)
   })
 
-  it('Logged in user with SLM_SECURITY_ANALYST role can not navigate to manual barcode entry page', () => {
-    cy.task('stubSignInWithRole_SLM_SECURITY_ANALYST')
+  it('Logged in user with SLM_ADMIN role can not navigate to manual barcode entry page', () => {
```

Comment:
Securiyt analyst role is gone as was never implemented. Which makes it easy to use the new role for hese tests.